### DISCUSSION TITLE
Custom variable or ending

### DISCUSSION THREAD

**Discussion by jmccay-work** [starship/starship#4882] [Q&A] [answered]
I am trying to create and ending to the prompt.  The character will still be the same each time, but I want to use the background color of the previous style as the foreground color.  How can I do this.  Thank you in advance.

**Comment by mickimnet** [comment, marked as answer]
AFAIK, there is currently no option in _Starship_ to get the color of the previous module.

**Comment by jmccay-work** [comment]
Thank you for your response.  It might be a nice feature in the future.
